### ISSUE TITLE
[BUG] crash on startup when starting with visualizer screen

### ISSUE THREAD

**Issue by PureArtistry** [ncmpcpp/ncmpcpp#432]
when starting with the ```startup_screen = visualizer``` config option or ```ncmpcpp -s visualizer``` it will crash with the following error:

![scrot_20201221-022532](https://user-images.githubusercontent.com/53883649/102732978-f489d980-4333-11eb-8d5c-676d960fd547.jpg)

if you start on any other screen and switch to the visualizer everything is fine

version: 0.9
os: arch linux

**Comment by xentec** [comment]
Same  crash, but without visualization. Here is a backtrace:
```
#0  0x00007ffff71df615 in raise () from /usr/lib/libc.so.6
#1  0x00007ffff71c8862 in abort () from /usr/lib/libc.so.6
#2  0x00007ffff741b86a in __gnu_cxx::__verbose_terminate_handler () at /build/gcc/src/gcc/libstdc++-v3/libsupc++/vterminate.cc:95
#3  0x00007ffff7427d9a in __cxxabiv1::__terminate (handler=<optimized out>) at /build/gcc/src/gcc/libstdc++-v3/libsupc++/eh_terminate.cc:48
#4  0x00007ffff7427e07 in std::terminate () at /build/gcc/src/gcc/libstdc++-v3/libsupc++/eh_terminate.cc:58
#5  0x00007ffff74280ae in __cxxabiv1::__cxa_throw (obj=obj@entry=0x55555581ad40, tinfo=tinfo@entry=0x5555557e7e18 <typeinfo for MPD::ClientError>, dest=dest@entry=0x5555556eea30 <MPD::ClientError::~ClientError()>) at /build/gcc/src/gcc/libstdc++-v3/libsupc++/eh_throw.cc:95
#6  0x00005555555a150f in MPD::Connection::checkConnection (this=<optimized out>) at /usr/include/c++/10.2.0/ext/new_allocator.h:89
#7  MPD::Connection::prechecks (this=<optimized out>) at mpdpp.cpp:885
#8  0x00005555556edad9 in MPD::Connection::prechecksNoCommandsList (this=0x5555557ebfe0 <Mpd>) at mpdpp.cpp:894
#9  MPD::Connection::GetDirectory (this=0x5555557ebfe0 <Mpd>, directory="/") at mpdpp.cpp:788
#10 0x00005555555bf01b in Browser::getDirectory (this=0x5555558d7340, directory="/") at screens/browser.cpp:507
#11 0x00005555555bfe2c in Browser::update (this=0x5555558d7340) at screens/browser.cpp:189
#12 0x00005555555bae95 in Browser::switchTo (this=0x5555558d7340) at screens/browser.cpp:168
#13 0x00005555555a67fe in main (argc=<optimized out>, argv=0x7fffffffe458) at ncmpcpp.cpp:160
```

**Comment by evanc577** [comment]
4709cc6b6d56f20f7e70dbc8fe0d1b8e2affc22b introduced the regression

git-bisect:

```
…/ncmpcpp on  HEAD (c51b27d) (BISECTING) [$?] 
❯ git bisect good
4709cc6b6d56f20f7e70dbc8fe0d1b8e2affc22b is the first bad commit
commit 4709cc6b6d56f20f7e70dbc8fe0d1b8e2affc22b
Author: Andrzej Rybczak <andrzej@rybczak.net>
Date:   Fri Dec 18 15:15:05 2020 +0100

    Deprecate visualizer_sync_interval

 CHANGELOG.md               |  1 +
 doc/config                 |  8 ++++++--
 doc/ncmpcpp.1              |  4 ----
 src/screens/visualizer.cpp | 18 ++++++++----------
 src/screens/visualizer.h   |  1 -
 src/settings.cpp           | 18 ++++++++++--------
 src/settings.h             |  3 +--
 7 files changed, 26 insertions(+), 27 deletions(-)
```

**Comment by arybczak** [comment]
:(

Fixed in 3d5d1f4d54a426cc4a4ca839b313597ab668a3ce and 40c40d8c7f011688b0014501be7fd5789fcf2569.
